src/App.js

```diff
@@ -1,24 +1,32 @@
-import logo from './logo.svg';
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import Crytpos from './Components/Crytpos';
+import Crypto from './Components/Crypto';
+import NavBar from './Components/NavBar';
+import DetailsHeader from './Components/DetailsHeader';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Routes>
+      <Route
+        path="/"
+        element={(
+          <>
+            <NavBar />
+            <Crytpos />
+          </>
+        )}
+      />
+      <Route
+        path="/crypto-details"
```

Comment:
- [ ] Kindly implement the unique routing as requested by the previous reviewer. Each crypto token on the home page should have its own unique route. Kindly observe the earlier request for a clue on how to go about this.

  You can consider using the useParams module from the react-router package as suggested.